### PR TITLE
Snails now utilize glasses offset system

### PR DESCRIPTION
**Pull request by Militaires**
image

:cl:
tweak: Snails now utilize glasses offset system
/:cl:

@Time-Green dumbie poopoohead....

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
lol

**Comment by tralezab** [comment]
oh my god

File: code/modules/surgery/organs/eyes.dm [hunk 1]
```diff
@@ -347,5 +347,4 @@
 	name = "snail eyes"
 	desc = "These eyes seem to have a large range, but might be cumbersome with glasses."
 	eye_icon_state = "snail_eyes"
-	icon_state = "snail_eyeballs"
-	no_glasses = TRUE
+	icon_state = "snail_eyeballs"
```
Comment (ShizCalev) [inline code/modules/surgery/organs/eyes.dm]: new line

**Comment by PKPenguin321** [comment]
Slurm Mackenzie hello

**Pull request merged**